pkg/api/v1alpha1/cluster_types.go

```diff
@@ -34,6 +34,7 @@ type ClusterSpec struct {
 	// +kubebuilder:validation:Optional
 	ExternalEtcdConfiguration *ExternalEtcdConfiguration `json:"externalEtcdConfiguration,omitempty"`
 	ProxyConfiguration        *ProxyConfiguration        `json:"proxyConfiguration,omitempty"`
+	ECRMirror                 *ECRMirror                 `json:"ecrMirror,omitempty"`
```

Comment:
I think RegistryMirror should be fine here, or RegistryMirrorConfiguration if we want to make naming similar to other configuration we have.